libcylinder/src/jwt/mod.rs

```diff
@@ -14,3 +14,504 @@
  * limitations under the License.
  * ------------------------------------------------------------------------------
  */
+
+//! Provides a set of APIs to both generate a JSON Web Token-like and cryptographically validate
+//! its contents.
+
+mod error;
+
+use std::collections::HashMap;
+
+use crate::{PublicKey, Signature, Signer, Verifier};
+
+pub use error::{JsonWebTokenBuildError, JsonWebTokenParseError};
+
+/// Builder for constructing the JWT string that would be included in request
+/// headers
```

Comment:
Do we need to mention request headers? Seems like an inappropriate level of detail here, especially since "headers" is used throughout this module in reference to the token's headers.

Reply:
Updated the doc to specify HTTP request headers.